package.json

```diff
@@ -6,9 +6,10 @@
     "Moritz von Hase",
     "Jonas Dohse <jonas@dohse.ch>",
     "Jan Lehnardt",
-    "Tim Ruffles <timruffles@gmail.com>"
+    "Tim Ruffles <timruffles@gmail.com>",
+    "Ashwani Agarwal <ashwani.a@outlook.com>"
   ],
-  "version": "0.6.0",
+  "version": "0.7.0",
```

Comment:
Please don't bump the npm version yet. I will do that in a separate commit before publishing to npm.